client/src/pages/HomePage/HomePage.component.jsx

```diff
@@ -62,7 +62,11 @@ const HomePage = ({ match }) => {
 
   return (
     <Flex direction="column" height="100%" className="home-page">
-      <PageTitle title={`${agency?.shortname.toUpperCase()} FAQ - AskGov`} />
+      <PageTitle
```

Comment:
nit: does it make sense to conditionally render it then?

Reply:
1. Does nit mean new in town?
2. No, I don't think so. We still need the page title to display the default titles set.

Reply:
nit means nitpick =)